.vscode/settings.json

```diff
@@ -10,8 +10,9 @@
         "editor.codeActionsOnSave": {
             "source.organizeImports": true,
         },
+        "editor.defaultFormatter": "ms-python.black-formatter",
     },
-    "python.formatting.provider": "black",
+    "python.formatting.provider": "none",
```

Comment:
I'm not sure why you changed this. Was that intentional? or is this now covered by what is added in line 13?

Reply:
you said something that you were not sure about this. So can me merge this or no?

Reply:
it works for me!